### DISCUSSION TITLE
Updating the upload background-image button text?

### DISCUSSION THREAD

**Discussion by HashimHaider** [GrapesJS/grapesjs#3988] [Q&A] [answered]
Hi guys,

So the background-image upload button has the text "Images". Is there any built-in property that'll allow the button text to be changed when adding the background-image property as part of our Sector? One solution is to do this through JS once the button is rendered, though that isn't ideal. 

![background-image](https://user-images.githubusercontent.com/81612773/144552446-a6d71a15-615c-4011-9c12-81e38a409726.png)
![button-text](https://user-images.githubusercontent.com/81612773/144552449-b0b60466-7041-43e1-b8d2-675408679dcc.png)

TIA!

**Comment by aliibrahim123** [comment, marked as answer]
you can change its i18n in the config
it is 
```
styleManager.fileButton
```

**Reply by ronaldohoch** [reply]
Hello @HashimHaider 
Can you share how did you do that?

**Reply by HashimHaider** [reply]
```
editor.I18n.addMessages({
        en: {
            styleManager: {
                fileButton: 'Upload',
            }
        }
    });
```

**Reply by ronaldohoch** [reply]
Thank you!

**Comment by HashimHaider** [comment]
Thank you!
